cmd/common/common.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"crypto/tls"
+	"net/url"
+
+	"github.com/pingcap/log"
+	"github.com/spf13/cobra"
+	"go.etcd.io/etcd/pkg/transport"
+	"go.uber.org/zap"
+
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/config"
+)
+
+// TODO: refactor tidb-dashboard main.go
+
+type DashboardCLIConfig struct {
```

Comment:
This configuration is only useful when starting the dashboard server instead of running Dashboard CLI tools. It is not very good to put it in `common`.